src/main/python/systemds/matrix/data_gen.py

```diff
@@ -29,6 +29,8 @@
 
 from systemds.operator import OperationNode
 from systemds.context import SystemDSContext
+from systemds.matrix import Matrix
+import numpy as np
 
 def full(sds_context: SystemDSContext, shape: Tuple[int, int], value: Union[float, int]) -> OperationNode:
```

Comment:
Also if you want to, then maybe change these data_gen functions to return matrix Objects, not Operation nods.